sandbox/grist/usertypes.py

```diff
@@ -225,17 +226,28 @@ class Bool(BaseColumnType):
   @classmethod
   def do_convert(cls, value):
     # We'll convert any falsy value to False, non-zero numbers to True, and only strings we
-    # recognize. Everything else will result in alttext.
+    # recognize.
```

Comment:
do_convert is a relatively "hot" function, it would be good to do any cachable work outside of it. Could you prepare the truthy/falsy values once rather than on each call?